src/client/index.js

```diff
@@ -287,7 +287,15 @@ export const signOut = async (args = {}) => {
   const res = await fetch(`${baseUrl}/signout`, fetchOptions)
   const data = await res.json()
   _sendMessage({ event: 'session', data: { trigger: 'signout' } })
-  window.location = data.url ?? callbackUrl
+  const finalLocation = new URL(data.url ?? callbackUrl)
```

Comment:
I wasn't sure myself if this would be a problem supporting IE11, or using it in Next.js would make sure that URL is polyfilled here?

Reply:
Nextjs includes URL polyfill. https://nextjs.org/docs/basic-features/supported-browsers-features

Reply:
I'm aware, but will those apply for node_modules as well? I remember we had to transpile next-auth further down at work to be able to run in IE11. Could you please confirm that this would work in IE11 as well by trying it out?